data/transactions/logic/eval.go

```diff
@@ -3522,10 +3548,52 @@ func (cx *EvalContext) stackIntoTxnField(sv stackValue, fs txnFieldSpec, txn *tr
 		txn.AssetReceiver, err = cx.availableAccount(sv)
 	case AssetCloseTo:
 		txn.AssetCloseTo, err = cx.availableAccount(sv)
-
-	// acfg likely next
-
-	// afrz seems easy but not high demand
+	// AssetConfig
+	case ConfigAsset:
+		txn.ConfigAsset, err = cx.availableAsset(sv)
```

Comment:
I considered this, too.  It seems ok to allow.  But I thought it would be better to teach developers that it's not necessary to do so, since it starts as 0. That saves 4 bytes or so of AVM code. I'm open to changing that opinion.

Reply:
Yeah that seems reasonable. I don't see a problem with that logic at all, but we may get developers asking why 0 isn't a valid input here.